x-pack/plugin/ml/qa/basic-multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/MlBasicMultiNodeIT.java

```diff
@@ -29,6 +28,8 @@
 
 public class MlBasicMultiNodeIT extends ESRestTestCase {
 
+    private static final String BASE_PATH = "/_ml/";
```

Comment:
Quite a few of the ML REST tests had a dependency on the ML main code for ONLY this base path reference in the tests.  Adding to each test like here is a bit redundant, but allows us to break that class path dependency and the REST path for ml is unlikely to change.

Reply:
Good call. And if it does, that's a breaking change anyhow.